### PR TITLE
SW-2490 Checkbox style fixes

### PR DESCRIPTION
**Pull request by karthikbtf**
- border radius and width are not fixed since it's an icon not css
- added comment in figma for ^
- more fixes to be done in web repo
- check out vercel

Table - Selectable ⋅ Storybook 2022-12-08 08-54-33

Checkbox - Top Align Long Label ⋅ Storybook 2022-12-08 08-54-17

Checkbox - Default ⋅ Storybook 2022-12-08 08-54-01

### CODE REVIEW

**Comment by karthikbtf** [comment]
need to fix disabled status

**Comment by karthikbtf** [comment]
> need to fix disabled status

done with an assumption, will see what @hannah-cai has to say on 0.7 opacity for disabled

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@terraware/web-components",
-  "version": "2.0.34",
+  "version": "2.0.35",
```
Comment (constanzauanini) [inline package.json:3]: I just merged my PR, so this need to be 36

File: src/components/Checkbox.tsx [hunk 2]
```diff
@@ -1,6 +1,31 @@
...
+  },
+  '&.Mui-checked > .MuiSvgIcon-root': {
+    fill: theme.palette.TwClrBgSelected,
+    color: theme.palette.TwClrBgSelected,
+  },
+  '&.MuiCheckbox-indeterminate > .MuiSvgIcon-root': {
+    fill: theme.palette.TwClrBgSelected,
+    color: theme.palette.TwClrBgSelected,
+  },
+  '&:focus > .MuiSvgIcon-root': {
```
Comment (constanzauanini) [inline src/components/Checkbox.tsx:21]: It looks like focus has a border radius in figma

Comment (karthikbtf) [reply src/components/Checkbox.tsx:21]: I didn’t add any border radius.. because the inner icon doesn’t have one. Added a comment in figma

**Review by constanzauanini — APPROVED** [review]

File: src/components/Checkbox.tsx (hunk 2 above)
Comment (karthikbtf) [reply src/components/Checkbox.tsx:21]: I added 2 more changes, removed some left margin and disabled animation on table checkboxes.

**Pull request merged**